plugins/hls-module-name-plugin/src/Ide/Plugin/ModuleName.hs

```diff
@@ -143,8 +160,8 @@ codeModuleName state nfp = runMaybeT $ do
   L (locA -> (RealSrcSpan l _)) m <- MaybeT . pure . hsmodName . unLoc $ pm_parsed_source pm
   pure (realSrcSpanToRange l, T.pack $ moduleNameString m)
 
--- traceAs :: Show a => String -> a -> a
--- traceAs lbl a = trace (lbl ++ " = " ++ show a) a
+newtype Log = ModuleNameLog T.Text deriving Show
```

Comment:
If possible, you should rather use Named constructors instead of a generic "Text" field here, since we can process logs later on, disable some log messages and allow others. In my opinion, something like this would be much cleaner:

```suggestion
data Log
  = LogSourcePaths [FilePath] -- ^ I am not sure right now what the type is, just assuming fp for now.
  | LogModulePaths [FilePath]
  | LogCanonicalisedPaths [FilePath]
```